vector/src/main/java/im/vector/widgets/WidgetsManager.java

```diff
@@ -85,7 +85,7 @@ public String getUIUrl() {
     /**
      * Widget error code
      */
-    public class WidgetError extends MatrixError {
+    public static class WidgetError extends MatrixError {
```

Comment:
I prefer this class not to be static, but defined in its own file

Reply:
moved